glad/generator/rust/templates/impl.rs

```diff
@@ -54,20 +67,31 @@ pub mod enumerations {
 }
 
 pub mod functions {
-    #![allow(non_snake_case, unused_variables, dead_code)]
+    #![allow({{ 'non_snake_case, ' if not options.mx }}unused_variables, dead_code)]
 
     use std;
     use std::mem;
-    use super::storage;
+    {{ 'use super::FnPtr;' if options.mx else 'use super::storage;' }}
     use super::types::*;
 
+    {{ 'pub struct Gl {' if options.mx }}
+    {% if options.mx %}
     {% for command in feature_set.commands %}
-    {{ template_utils.protect(command) }} #[inline] pub unsafe fn {{ command.name|no_prefix }}({{ command|params }}) -> {{ command.proto.ret|type }} { mem::transmute::<_, extern "system" fn({{ command|params('types') }}) -> {{ command.proto.ret|type }}>(storage::{{ command.name|no_prefix }}.ptr)({{ command|params('names') }}) }
+    {{ '    ' }}{{ template_utils.protect(command) }} pub _{{ command.name|no_prefix }}: FnPtr,
```

Comment:
`{{ '    ' }}` can just indent by that much ;)

Reply:
Heh, cool! Didn't know that :D Fixed.